languages/java/INSTALL.md

```diff
@@ -40,6 +40,12 @@ jenv shell 17
 ./gradlew build
 ```
 
+### Windows
+
+Download JDK for windows here: https://www.oracle.com/in/java/technologies/downloads/#java17
```

Comment:
We shouldn't recommend oracle distributions, it's EOL for non commercial support. The macOS instructions uses eclipse OpenJDK which is supported until 2027

Reply:
Just removed the oracle link, thanks Oscar!